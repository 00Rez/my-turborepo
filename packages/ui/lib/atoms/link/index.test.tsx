import { describe, it, expect } from "vitest";
import { Link } from ".";
import { render } from "@testing-library/react";

describe(Link.name, () => {
  it("renders", () => {
    const { container } = render(<Link href="https://example.com">Example Link</Link>);
    expect(container).toMatchSnapshot();
  });
});
