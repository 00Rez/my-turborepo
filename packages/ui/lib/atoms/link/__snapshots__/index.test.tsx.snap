// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Link > renders 1`] = `
<div>
  <a
    href="https://example.com"
  >
    Example Link
  </a>
</div>
`;
